packages/builder/src/types/bindings.ts

```diff
@@ -0,0 +1,8 @@
+import { CompletionContext, Completion } from "@codemirror/autocomplete"
+
+export type BindingCompletion = (context: CompletionContext) => {
```

Comment:
These types are only used in the builder, and they have dependencies that are not used anywhere else. It makes more sense to live in here that in the `types` package